### ISSUE TITLE
ds4 windows not installing

### ISSUE THREAD

**Issue by trickster1129** [Jays2Kings/DS4Windows#104]
I am new to everything pc and I downloaded ds4windows, I chose appdata by mistake when I was asked where should to save ds4windows. Now, I am trying to run the exe and whenever I click number 1 which is to install ds4windows driver, I keep getting install failed and the virtual bus driver folder is opening by itself. I hope someone can help me with this. thanks in advance

**Comment by ottercrazyone** [comment]
I had same issue, go to window tab right click, click Run, type %appdata%, open it. Right click on ds4windows and delete. Empty trash folders. Also delete all installed ds4windows in your downloads or where ever else they may show. Reinstall everything again, this time don't choose appdata. It may not work the first time but try it a second time. If not reset pc And try again it worked for me

**Comment by Rose2234** [comment]
yep it's still an issue and the dev does not care anymore all he gives out is excuses like"i have been too busy" cliche excuses. i am using Windows 10 Enterprise x64 latest build

**Comment by rhx3030** [comment]
go to https://github.com/Ryochan7/DS4Windows/releases
it's a great solution
